### ISSUE TITLE
Large vs Small Number Entry Fields Styling Notes

### ISSUE THREAD

**Issue by marcaaron** [marcaaron/ww-calc-2.0#18]
via Jen... The number entry fields specifically for numbers that are only in the double digits (number of adults, size of household, etc) have excessively large widths.

Another issue brought up is that where long numbers are to be inputted there are unnecessary up/down arrows. 

These are generated per browser and exist because currently both larger numbers (e.g. income) and smaller numbers (e.g. size of household) are treated as the same type of 'number' input and we should really be treating them differently. My suggestion would be to change the questions with longer numbers to a custom inputType that no longer uses the `<input type="number">` tag but `<input type="text">` instead.

Since these fields were in fact initially `type="text"` fields not `type="number"` fields the classNames will need to be updated. Currently an inputType of number is confusingly applied a className of 'text-box' so I would propose the following:

**Steps:**

- Add new `inputType` "text" to the render method within `Question.js` component to handle larger numbers as text fields with className of 'text-box'
- Change className of the `inputType === 'number'` render to something like `number-box`
- Edit number box style to have a smaller width.
- Update the `questions.js` file so that any question object receiving a larger number has an inputType property value of 'text'
- Check for responsive design quirks 
- Check for any unexpected behavior that might result from switching `input type="number"` to `input type="text"`

**Comment by dentemple** [comment]
My recommendation would be to treat currency values separately from regular numeric values.

So, `inputType` would have separate options for `number` and `currency`, and then we can create separate functions to handle the different types of values.

**Comment by marcaaron** [comment]
Hmm, that's a good idea. I didn't see this and went ahead and tooled around. Maybe take a look at my changes?

Any question looking for currency is now set to inputType 'text' which now that I'm thinking about it isn't very accurate - but I did switch to using text boxes for those larger numbers so that kind of makes sense.

In order to get rid of the up/down keys on larger numbers I changed those to text inputs and ran it through some extra form validation to block non-numbers + meta keys (a step I was avoiding before by just going with input type number for everything).

Right now, I've also got a '$' icon rendering depending on whether a unit is provided via the question. We're really only looking for one unit here so it works for our case, but could certainly be more flexible.

We could use the same `this.props.unit` hook to take care of #20 or come up with something better.
